### DISCUSSION TITLE
Keygen sh Zapier Stripe Automation

### DISCUSSION THREAD

**Discussion by axen2u** [keygen-sh/community#147] [Q&A] [answered]
I’m trying to automate licensing with keygen sh with stripe using Zapier. When I made a payment I am getting 400 badway error keygen sh create licence node.

When I am checking the logs on keygen sh side it has written the id is a string not uuid string. Following json is the error body.

 `   {
      "meta": {
        "id": "5ac0f653-4351-415c-aa0d-7661c8da256e"
      },
      "errors": [
        {
          "title": "Bad request",
          "detail": "type mismatch (received string expected UUID string)",
          "source": {
            "pointer": "/data/relationships/owner/data/id"
          }
        }
      ]
    }`

**Comment by ezekg** [comment, marked as answer]
It sounds like you're trying to create a license with a user, and you're using an email address where you should be using a user ID. This isn't allowed. You will need to use the Find User action to lookup the user by their email, and then use the resulting `Data.Id` in the license's user relationship.

**Reply by axen2u** [reply]
Thanks @ezekg
